### PR TITLE
added command to find vhd location

### PR DESCRIPTION
**Pull request by mvaferreira**
Added powershell command to find the VHD location and the missing 'diskpart' command.

### CODE REVIEW

File: WSL/wsl2-ux-changes.md [hunk 1]
```diff
@@ -54,12 +54,18 @@ The picture below shows an example of this by connecting to a python server runn
 WSL 2 stores all your Linux files inside of a VHD that uses the ext4 file system. This VHD automatically resizes to meet your storage needs. This VHD also has an initial max size of 256GB. If your distro grows in size to be greater than 256GB you will see errors stating that you've run out of disk space. You can fix these by expanding the VHD size. Instructions on how to do so are below:
 
 1. Terminate all WSL instances using the `wsl --shutdown` command
```
Comment (mscraigloewen) [inline WSL/wsl2-ux-changes.md]: Can you add a note after this set of instructions (in other words the line above ## WSL 2 will use some memory... ) that states this:

Please note: In general do not modify, move, or access the WSL related files located inside of your AppData folder using Windows tools or editors. Doing so could cause your Linux distro to become corrupted.

**Comment by mscraigloewen** [comment]
Added a slight change, and once that's in I'll merge it!

**Comment by mvaferreira** [comment]
Done, note updated with warning regarding changes in AppData contents.

**Comment by mscraigloewen** [comment]
Thanks for your submission!

**Pull request merged**